### DISCUSSION TITLE
Open Stream in Browser / Open in Streamlink included in right click menu on Tabs

### DISCUSSION THREAD

**Discussion by damnjuhl** [Chatterino/chatterino2#3235] [Ideas]
Currently this option is available when clicking the title at the top of the window, I think it would be beneficial if this option was also available when clicking on the different stream tabs. Right now, the right click context menu on the tabs just does management of the tabs and no interaction with the content of those tabs.

![aqUguog4M8](https://user-images.githubusercontent.com/15189394/133011833-1766dd3b-c5ca-4cfe-92c0-c8084db66320.png)
![3GS6yqnAUE](https://user-images.githubusercontent.com/15189394/133011911-4a2e0593-bdd4-4c88-ab77-c8430b6b3c65.png)

**Comment by zneix** [comment]
What if a single tab has multiple (or zero) splits within in?
What if a tab contains non twitch channels (special chatterino channels like /mentions or /whispers, IRC channels, or special debugging channels, like "$$$")?

You idea may have good intentions, but it's never going to work because of scenarios I've explained above. It's straight up impossible to consistently access exactly - only one split from its parent tab.
And even if it's possible in some cases there's ugly code duplication coming in that place which isn't the best practice imho.
